api/aggregators/searchAggregator.js

```diff
@@ -174,7 +174,9 @@ const generateExpArray = async (field, roles, schemaName) => {
     console.log("queryString:", queryString);
 
     await Promise.all(Object.keys(queryString).map(async item => {
-      const entry = queryString[item];
+      // make sure the entry isn't URL encoded. Also, shift any ampersand
+      // values from %26 to &
+      const entry = decodeURI(queryString[item]).replace('%26', "&");
```

Comment:
We should be able to use `decodeURIComponent` which handles the ampersand. Based on this stackoverflow [post](https://stackoverflow.com/questions/747641/what-is-the-difference-between-decodeuricomponent-and-decodeuri)

Reply:
Yep, decodeURIComponent appears to do the trick. I've swapped it out